ingestion/src/metadata/orm_profiler/api/workflow.py

```diff
@@ -315,7 +315,7 @@ def filter_entities(self, tables: List[Table]) -> Iterable[Table]:
                     table.databaseSchema.name,
                 ):
                     self.source_status.filter(
-                        table.databaseSchema.fullyQualifiedName,
+                        f"Schema pattern not allowed: {table.fullyQualifiedName.__root__}",
```

Comment:
what's the goal of adding the "Schema pattern not allowed" here?

Reply:
agree on the __root__ though

Reply:
I am thinking it is clearer for the user point of view -- this table was filtered because of schema pattern not allowed. wdyt?